### ISSUE TITLE
back arrow between two navigationView

### ISSUE THREAD

**Issue by ishigo1987** [eclipsesource/tabris-js#1243]
Hello i have a question.
let assume that we have two navigationView A and B
if from navigationView A i click on an button and i go to the navigationView B
is it possible to have a back arrow(back button) to return to navigationView A?
if is possible how can i do that?

**Comment by ralfstx** [comment]
You can use the [backnavigation](https://tabrisjs.com/documentation/latest/api/app#backnavigation) event on `tabris.app` to hide one NavigationView and show the other.

**Comment by ishigo1987** [comment]
ok i can use the event but the user can't see the back button at the top of the page is it right?

**Comment by ishigo1987** [comment]
or i can simulate this button with an image on the header(a back button image)

**Comment by ralfstx** [comment]
Right, that would not work with the back button in the navigation view. I think even if you faked it, you wouldn't end up with a native UX.

You can have more than one NavigationView in your app. But if you want them to appear as a *single* one, you should rather implement just a single NavigationView and replace the page stacks instead. You can copy the page stack from `navigationView.pages()`, remove pages using `page.detach()` and restore them later using `page.appendTo(navigationView)`.

**Comment by ishigo1987** [comment]
Have you ever seen the animation in android 5.0 or 5.1 in the message application?
we have a list of messages and when we clicks on a message the view of this message appears by making a translation from bottom to the top.
It is not only the page that translates but the whole navigation view (header + page) actually my questions above is because I want to do the same animation system.it's look like a stack of navigationView with a back button
Is it possible with tabrisJS?

**Comment by ishigo1987** [comment]
is anyone there?

**Comment by mpost** [comment]
@ishigo1987 The animation you are describing is possible. The whole navigation view itself can be animated it the way you described.

For the back navigation interception you can use the `backnavigation` event as pointed out before.

**Comment by ishigo1987** [comment]
ok thank you

**Comment by ralfstx** [comment]
This issue appears resolved. Please reopen if it's not.
